pages/getting-started/content/de/guide.md

```diff
@@ -0,0 +1,170 @@
+# Hallo 👋
+
+Wir werden dir `glamorous` beibringen, indem du das Tuturial bearbeitest,
+welches du gerade liest 😱
+
+Um anzufangen haben wir links bereits ein paar Komponenten für dich bereitgestellt,
+also musst du nur deren Stile anpassen, um die Art und Weise zu verändern,
```

Comment:
`also musst du nur deren Stile anpassen, um die Art und Weise, wie dieser Guide aussieht und sich bewegt, zu verändern.` instead of `also musst du nur deren Stile anpassen, um die Art und Weise zu verändern,`